### ISSUE TITLE
Help insufficient

### ISSUE THREAD

**Issue by ufechner7** [MakieOrg/Makie.jl#3730]
For example:
```
help?> Menu.alignmode
  Default value: Inside()

  The alignment of the menu in its suggested bounding box.
```
The default value is mentioned, but nowhere a hint which other values are allowed...

This is is the same for many other attributes.

**Comment by asinghvi17** [comment]
I wonder if one could define hooks for extra, common info when displaying docs in the REPL.  Maybe by dispatching on Val{property}()?

**Comment by ufechner7** [comment]
> I wonder if one could define hooks for extra, common info when displaying docs in the REPL. Maybe by dispatching on Val{property}()?

Good idea, but in the html documentation it should be possible to click on any of the attributes to get a list of valid values. Would that be possible with the currently used documentation system?
